code/LINDA/LINDA_turf_tile.dm

```diff
@@ -305,17 +305,24 @@
 
 
 /atom/movable/var/pressure_resistance = 5
+/atom/movable/var/throw_pressure_limit = 15
 /atom/movable/var/last_forced_movement = 0
 
 /atom/movable/proc/experience_pressure_difference(pressure_difference, direction)
 	if(last_forced_movement >= air_master.current_cycle)
 		return 0
 	else if(!anchored && !pulledby)
-		if(pressure_difference > pressure_resistance)
+		if(pressure_difference >= throw_pressure_limit)
+			var/general_direction = get_edge_target_turf(src,direction)
+			if(ismob(src)) to_chat(src, "<span class='userdanger'>The pressure sends you flying!</span>")
```

Comment:
Ah, I worded that poorly, didn't I? By "one-line code block", I meant the spawn's block of code being on the same, one line as the `spawn` statement. This:
```
spawn throw_at(...)
```
should've looked like this (if you hadn't replaced it with `throw_at_fast`):
```
spawn()
  throw_at(...)
```
The `spawn step(...)` below does the same thing; it makes it less clear that it's spawning off execution of a separate block of code.